co2eq/purchase.js

```diff
@@ -33,7 +33,9 @@ import {
   PURCHASE_CATEGORY_ENTERTAINMENT_LIQUOR_STORE,
   ACTIVITY_TYPE_MEAL,
   ACTIVITY_TYPE_TRANSPORTATION,
-  TRANSPORTATION_MODE_CAR,
+  TRANSPORTATION_MODE_ICE_CAR,
```

Comment:
we should introduce a VEHICLE_TYPE instead of more TRANSPORTATION_MODE

Reply:
Hello @corradio @ovbm , this is a bit too complicated for me. Can you setup the structure and I can then help you move things around?

Reply:
I'll help

Reply:
@corradio any update on this one?

Reply:
Not yet - we need to decide on how to merge nested transportation modes into footprint.yml I think.